### PR TITLE
Upgrade/9.0

### PR DESCRIPTION
**Pull request by garvit-rajput**

### CODE REVIEW

File: projects/ui/src/lib/oui/autocomplete/autocomplete.spec.ts [hunk 1]
```diff
@@ -85,8 +85,8 @@ class SimpleAutocomplete implements OnDestroy {
 
   @ViewChild(OuiAutocompleteTrigger, { static: true })
   trigger: OuiAutocompleteTrigger;
-  @ViewChild(OuiAutocomplete, { static: false }) panel: OuiAutocomplete;
-  @ViewChild(OuiFormField, { static: false }) formField: OuiFormField;
+  @ViewChild(OuiAutocomplete) panel: OuiAutocomplete;
```
Comment (mderazon) [inline projects/ui/src/lib/oui/autocomplete/autocomplete.spec.ts:88]: @garvit-rajput these are being done automatically by the angular cli ?
Did you make any manual changes that weren't covered by the cli upgrade tool ?

File: projects/ui/tsconfig.lib.prod.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```
Comment (mderazon) [inline projects/ui/tsconfig.lib.prod.json:4]: We're disabling Ivy ?

File: tsconfig.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
     "declaration": false,
-    "module": "es2015",
+    "module": "esnext",
```
Comment (mderazon) [inline tsconfig.json:8]: What's the implications here ?

File: projects/ui/src/lib/oui/autocomplete/autocomplete.spec.ts (hunk 1 above)
Comment (garvit-rajput) [reply projects/ui/src/lib/oui/autocomplete/autocomplete.spec.ts:88]: I haven't made any manual changes other than updating dependency in package json for some packages

**Pull request closed without merging**